### DISCUSSION TITLE
When will RISC-V be officially stated as supported on https://github.com/memcached/memcached/wiki/Hardware

### DISCUSSION THREAD

**Discussion by Tenorotti** [memcached/memcached#1172] [General]
Hi doramando, when will Memcached have RISC-V listed as supported on this page? https://github.com/memcached/memcached/wiki/Hardware

Thanks in advance,

Isaac

**Comment by dormando** [comment]
Hi. I'm not sure. I did acquire some risc-v hardware but haven't had the time to set it up.

I have heard that it builds and runs just fine, but I don't have official buildbots for it yet so I can't CI test it.

Have you tried it or what's your intention here?
